packages/grafana-prometheus/src/components/monaco-query-field/promql.ts

```diff
@@ -0,0 +1,247 @@
+// The MIT License (MIT)
+//
+// Copyright (c) Celian Garcia and Augustin Husson @ Amadeus IT Group
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
```

Comment:
Have you confirmed if this will be stripped when bundling? Not sure what implications it has. 🤔

Reply:
I can confirm it is stripped when bundling. I know having the MIT license allows any further usage to copy the code and change the license though, so when it becomes AGPL in the library, that should be ok.